more random mappings

moehreag opened the pull request:
This time I should have managed to not map things others already mapped!

This will probably be the last PR for now unless I find more things I know the names of (or am able to figure them out).

Again, if things need to be changed (for whatever reason) just yell at me.

moehreag committed:
more random mappings

moehreag committed:
fix name to respect conventions

EnnuiL commented on mappings/net/minecraft/UrlConstants.mapping:
```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_goxaugqo net/minecraft/UrlConstants
```
ohhh, that's a neat alternative pattern that `ScreenTexts` could be renamed to; Hm, I do wonder if `TextConstants` would fit though;
anyway, yeah, i don't like `ScreenTexts` and Mojmap's `CommonComponents` is close to being better; therefore, it should be renamed and it should share a theme with this

oh, about the package, hm, i do wonder if it should be at `util`

moehreag replied:
it's at net.minecraft.util now (even though SharedConstants isn't)

EnnuiL commented on mappings/net/minecraft/client/util/PlayerKeyPairManagerImpl.mapping:
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_mdilqcsv net/minecraft/client/util/PlayerKeyPairManagerImpl
+	FIELD f_thgwzahx PROFILEKEYS Ljava/nio/file/Path;
```
hm, I don't know the context, but I feel like there should be an underline:
```suggestion
	FIELD f_thgwzahx PROFILE_KEYS Ljava/nio/file/Path;
```

moehreag replied:
I named it like this because the path string is "profilekeys" (all lowercase), but I suppose it doesn't really matter

moehreag commented:
I'll fix the name causing the check to fail tomorrow, it's too late right now

moehreag committed:
fix checks + add Ennui's suggestions

I won't rename the ScreenTexts class because that change might need some more discussion in before

moehreag committed:
clarify name

eristhea commented on mappings/net/minecraft/block/TorchFlowerBlock.mapping:
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_ujhirzat net/minecraft/block/TorchFlowerBlock
```
just a nitpick about capitalisation, since the identifier for this block is `minecraft:torchflower` I'd prefer we went with `TorchflowerBlock` as one word
make sure to add it to `minecraft_specific_words.txt`!

eristhea commented on mappings/net/minecraft/block/TorchFlowerBlock.mapping:
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_ujhirzat net/minecraft/block/TorchFlowerBlock
+	FIELD f_qzqdcidr SHAPE [Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_thxmbdfh AGE_PROPERTY Lnet/minecraft/unmapped/C_vltzvhxi;
```
the property suffix is redundant here
```suggestion
	FIELD f_thxmbdfh AGE Lnet/minecraft/unmapped/C_vltzvhxi;
```

eristhea commented on mappings/net/minecraft/client/gui/AbuseReportScreen.mapping:
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_eagjljtr net/minecraft/client/gui/AbuseReportScreen
```
wrong package!
```suggestion
CLASS net/minecraft/unmapped/C_eagjljtr net/minecraft/client/gui/screen/AbuseReportScreen
```

eristhea commented on mappings/net/minecraft/world/gen/feature/TreeConfiguredFeatures.mapping:
```diff
@@ -7,6 +8,7 @@ CLASS net/minecraft/unmapped/C_fxqldlqj net/minecraft/world/gen/feature/TreeConf
 	FIELD f_dsunpqgn MANGROVE Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_eyldriwt ACACIA Lnet/minecraft/unmapped/C_xhhleach;
+	FIELD f_fboflfct CHERRY_BEES_005 Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_gnvxdkri BIRCH_BEES_005 Lnet/minecraft/unmapped/C_xhhleach;
```
completely unrelated but remember when this field was `BICH_BEES_005` for like two years

eristhea reviewed (changes requested):
amazing work, I *love* the amount of new classes mapped here!

moehreag committed:
address rai's requested changes

Mysterious-Dev commented on mappings/net/minecraft/data/server/loot_table/DesertWellLootTableGenerator.mapping:
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_uakojvce net/minecraft/data/server/loot_table/DesertWellLootTableGenerator
```
I'm not sure for `DesertWellLootTableGenerator` (we have Desert Well Archaeology & DesertPyramid Archaeology). Maybe `DesertArchaeologyLootTableGenerator` instead.

```suggestion
CLASS net/minecraft/unmapped/C_uakojvce net/minecraft/data/server/loot_table/DesertArchaeologyLootTableGenerator
```

moehreag replied:
sure!

Mysterious-Dev commented on mappings/net/minecraft/loot/OneTwentyLootTables.mapping:
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/unmapped/C_hcsgzkdg net/minecraft/loot/OneTwentyLootTables
```
That's not exactly all "OneTwenty" loot tables, it's just for archaeology.

```suggestion
CLASS net/minecraft/unmapped/C_hcsgzkdg net/minecraft/loot/OneTwentyArchaeologyLootTables
```

moehreag replied:
alright, though I wonder how Mojang came up with the mojmap name then (It's `UpdateOneTwentyBuiltInLootTables`).

moehreag committed:
Update mappings/net/minecraft/loot/OneTwentyLootTables.mapping

Co-authored-by: Mysterious_Dev <40738104+Mysterious-Dev@users.noreply.github.com>

moehreag committed:
Update mappings/net/minecraft/data/server/loot_table/DesertWellLootTableGenerator.mapping

Co-authored-by: Mysterious_Dev <40738104+Mysterious-Dev@users.noreply.github.com>

github-actions[bot] committed:
Merge branch '1.19.4-rc3' into 1.19.4-rc2-random-things

actions-user committed:
Drop invalid mappings
